### ISSUE TITLE
Log timestamp not match system timezone

### ISSUE THREAD

**Issue by Mosney** [AdguardTeam/AdGuardHome#1234]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [Y] I am running the latest version
- [Y] I checked the documentation and found no answer
- [Y] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * 0.99.3
* **How did you setup DNS configuration:**
  * Router
* **If it's a router or IoT, please write device model:**
  * amd64 device 
* **Operating system and version:**
  * OpenWrt 19.07rc2

### Expected Behavior

Log record local time (UTC+8) as like query log timestamp in WebUI 
### Actual Behavior

It looks like use UTC time and not localized
### Screenshots

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/22477230/70267510-926a3f00-17d9-11ea-8405-d0f47bcc1f1d.png)
</details>

**Comment by ameshkov** [comment]
@ArtemBaskal @IldarKamalov  any idea?

**Comment by rufengsuixing** [comment]
on my openwrt router R9.11.23  kernel 4.14.155 ,runtime log time is utc time,querylog time is the right local time

**Comment by norikohxb** [comment]
I am in China the DNS query LOG time is different with my time zone.
so it is hard for me to  get DNS history.

**Comment by norikohxb** [comment]
I guest this was becaused when you have to download the AD file
wget https://140.82.113.3/AdguardTeam/AdGuardHome/releases/download/v0.99.3/AdGuardHome_linux_amd64.tar.gz
you have to swith to your gateway to a router which has V2ray or SSR and then when installing ADG it will treat your computer as an USA time zone computer. so all your AdGuardHome time will be updated to USA time.
now how to change it back to Chinese time Zone ,or how to change time zone of ADG

**Comment by Mosney** [comment]
> I guest this was becaused when you have to download the AD file
> wget https://140.82.113.3/AdguardTeam/AdGuardHome/releases/download/v0.99.3/AdGuardHome_linux_amd64.tar.gz
> you have to swith to your gateway to a router which has V2ray or SSR and then when installing ADG it will treat your computer as an USA time zone computer. so all your AdGuardHome time will be updated to USA time.
> now how to change it back to Chinese time Zone ,or how to change time zone of ADG

Any evidence? I think it no sense. Just a download and extract file.

**Comment by norikohxb** [comment]
because it use my VPS (v2ray server) to download .and to this computer all connection with non Chinese address all use VPS's USA ip. maybe when AdGuardHome installing it need to connect to some URL or IP to  check and becuase it use v2ray USA ip to complete this task and then It think this software inernal time should be USA time.

**Comment by rufengsuixing** [comment]
no software get timezone from ip ，it make no sense

**Comment by norikohxb** [comment]
on how can i correct my log time. if my local time is 02:00 , then my AdGuardHome Log  time is 10.00
(10.00-2.00=8.00) how to correct it .
i need a correct time in Log system.

**Comment by Mosney** [comment]
So this issue has been resolved?

**Comment by norikohxb** [comment]
yes dear the mistake is casued by esxi system time problem
after change it's time. everything is ok now

**Comment by Mosney** [comment]
Thx a lot!

**Comment by zd3sf** [comment]
I don't think this issue is resolved. My logs are ahead by 5 hours, so basically UTC. Running on ddwrt using a OPKG .ipk that I compiled my self from AdGuardHome_linux_armv7.tar.gz adguardhome-go_v0.107.79. In my YAML, 
```
log:
  enabled: true
  file: ""
  max_backups: 0
  max_size: 100
  max_age: 3
  compress: false
  local_time: true
  verbose: false
```

Output of `tr '\0' '\n' < /proc/3993/environ | grep '^TZ='` 3993 is AdGuardHome PID  was:
```
TZ=CST6CDT,M3.2.0,M11.1.0
```
